src/tool/subcommands/api_cmd/test_snapshots.txt

```diff
@@ -114,6 +114,12 @@ filecoin_stateminersectors_1743095345529547.rpcsnap.json.zst
 filecoin_statenetworkname_1737546933390169.rpcsnap.json.zst
 filecoin_statenetworkversion_1737546933390184.rpcsnap.json.zst
 filecoin_statereadstate_1737546933390259.rpcsnap.json.zst
+filecoin_statereadstate_1747857537534929.rpcsnap.json.zst
+filecoin_statereadstate_1747857537535135.rpcsnap.json.zst
+filecoin_statereadstate_1747857537535290.rpcsnap.json.zst
+filecoin_statereadstate_1747857537534606.rpcsnap.json.zst
+filecoin_statereadstate_1747857537535026.rpcsnap.json.zst
+filecoin_statereadstate_1747857537535209.rpcsnap.json.zst
```

Comment:
We really should have a way of commenting on what those snapshots are. Obviously not something to be done in this PR, but what we could do here is to add something to the name,  e.g., `statereadstate_account_1234.rpcsnap.json.zst`, or something like this.